blocks/banner/banner.js

```diff
@@ -17,7 +17,7 @@ export default async function decorate(block) {
 
   block.appendChild(bannerTextWrapper);
 
-  const imageUrl = document.querySelector('.banner-text-wrapper picture img').src;
-
-  block.style.backgroundImage = `url(${imageUrl})`;
+  const image = block.querySelector('.banner-text-wrapper picture img');
+  block.style.backgroundImage = `url(${image.src})`;
```

Comment:
Maybe using `image.currentSrc` would be better in this case so you're not reloading another image eg the default png image instead of the properly resized webp image unnecessarily see https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/currentSrc

Reply:
now uses currentSrc